Documentation/gettingstarted/dns.rst

```diff
@@ -30,10 +30,11 @@ In line with our Star Wars theme examples, we will use a simple scenario where t
 
 .. parsed-literal::
 
-    $ kubectl create -f \ |SCM_WEB|\/examples/kubernetes-dns/dns-sw-app.yaml
-    $ kubectl get po
-    NAME                             READY   STATUS    RESTARTS   AGE
-    pod/mediabot                     1/1     Running   0          14s
+   kubectl create -f \ |SCM_WEB|\/examples/kubernetes-dns/dns-sw-app.yaml
+   kubectl wait pod/mediabot --for=condition=Ready
```

Comment:
[For what it's worth, I'm not entirely convinced about removing the prompts, as some processing will still be needed after copying to remove the output from the commands. I'm working on adding a new button to copy just the commands starting with the prompt symbol, but excluding the prompt from the copy itself. But this is not directly relevant to this PR, so OK for the change.]

Reply:
Yes, this won't work using the "copy all" button anyhow. The prompt removal is more so that one may copy/paste by highlighting a specific line: at the moment it also copy/pastes the `$`.
![image](https://user-images.githubusercontent.com/4659919/118502443-ffea0100-b729-11eb-9426-556b15a75db4.png)